### DISCUSSION TITLE
ND Freezes

### DISCUSSION THREAD

**Discussion by krazyj83** [flybywiresim/aircraft#2729] [Help] [answered]
Not sure if this is whats meant by CPT ND turns off, but I found that using the latest version with the custom FBW mod in 1,12,13,0 the ND tends to freeze and eventually have a red compass arc 

turning the display on/off or changing it to rose does not relieve the issue, anyone else experience this ?

**Comment by krazyj83** [comment, marked as answer]
following a bit of troubleshooting with the help of a discord member, it seems to be related to a conflict with the Working Title mods for CJ4

**Reply by Mark16161** [reply]
I had the same problem and it went away after I removed that CJ4 mod from the Community folder. Go figure ...

**Reply by hu277** [reply]
I haven't installed the CJ4 mod but get the same freezing error of the ND as mentioned above.The error occurs when using a direct to in the FMS.
